upload/common/validation_event.py

```diff
@@ -10,10 +10,22 @@
 
 class ValidationEvent:
 
+    @classmethod
+    def load(cls, db_id):
+        db = UploadDB()
+        prop_vals_dict = db.get_pg_record("validation", db_id)
+        return cls(
+            validation_id=db_id,
+            job_id=prop_vals_dict['job_id'],
+            file_id=prop_vals_dict['file_id'],
+            status=prop_vals_dict['status'],
+            docker_image=prop_vals_dict['docker_image']
+        )
+
     def __init__(self, **kwargs):
         self.id = kwargs["validation_id"]
         self.job_id = kwargs["job_id"]
-        self.file_id = kwargs["file_id"]
+        self.file_id = kwargs.get("file_id")
```

Comment:
I'm probably missing something here. Why would we init a validation event without a file id?

Reply:
Ah nevermind. This is from the docker image which doesn't actually update the event db record